sparse/coo.py

```diff
@@ -1954,6 +1894,20 @@ def tril(x, k=0):
     return COO(coords, data, x.shape, x.has_duplicates, x.sorted)
 
 
+def _nary_match(*arrays):
```

Comment:
I'm not able to quickly figure out what this function does.  Can I ask you for a small docstring?  If possible I find small example sections in docstrings to be very helpful when learning codebases that others have written.

Reply:
Whoops, must have missed that one.

Reply:
It looks like this function is no longer used.  Delete?